omnij-rpc/src/main/java/foundation/omni/json/conversion/AddressBalanceEntriesDeserializer.java

```diff
@@ -0,0 +1,50 @@
+package foundation.omni.json.conversion;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.deser.BeanDeserializer;
+import com.fasterxml.jackson.databind.deser.std.ContainerDeserializerBase;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.fasterxml.jackson.databind.type.SimpleType;
+import foundation.omni.rpc.AddressBalanceEntries;
+import foundation.omni.rpc.AddressBalanceEntry;
+import foundation.omni.rpc.BalanceEntry;
+
+import java.io.IOException;
+
+/**
+ *
+ */
+public class AddressBalanceEntriesDeserializer extends StdDeserializer<AddressBalanceEntries> {
```

Comment:
Figuring out how to do this via limited Jackson documentation took me more time than I'd like to admit.